config/end2end.php

```diff
@@ -1,10 +0,0 @@
-<?php
-
-return [
-    'api_url' => 'http://end2end--gateway.elife.internal',
```

Comment:
`api_url` would still be needed as config? Although the file should be provided by the formula